pg_query_normalize.c

```diff
@@ -358,3 +358,14 @@ PgQueryNormalizeResult pg_query_normalize(char* input)
 
 	return result;
 }
+
+void pg_query_normalize_free(PgQueryNormalizeResult result)
+{
+  if (result.error) {
+    free(result.error->message);
+    free(result.error->filename);
+  }
+
+  free(result.error);
```

Comment:
Stylistically speaking, I'm wondering if this shouldn't be inside the if clause.

`free()` doesn't do anything on a NULL pointer, but it'd feel more logical to me if its in there.